website/pages/docs/plugins/sources/aws/configuration.md

```diff
@@ -234,3 +236,38 @@ This is used to specify one or more accounts to extract information from. Note t
 - `skip_member_accounts` ([]string)
 
   List of OU member accounts to skip. This is useful in conjunction with `organization_units` if there are accounts under the selected OUs that should be ignored.
+
+## Advanced Configuration
+
+### Skip Tables
+
+AWS has tables that may contain many resources, nested information, and AWS-provided data.  These tables may cause certain syncs to be slow due to the amount of AWS-provided data and may not be needed.  Below is a reference configuration of skip tables, where certain tables are skipped.  We recommend validating if the data contained in the tables below are necessary and to ensure syncing from required tables.
+
+```yaml
+kind: source
+spec:
+  # Source spec section
+  name: aws
+  path: cloudquery/aws
+  version: "v17.2.0"
+  tables: ["*"]
```

Comment:
I think listing the typically slow tables is fine, but we should also recommend against the practice of using `*`, because it can cause syncs to slow down over time as new tables are added.

Reply:
Yeah agree. I think the recommendation should be just specifying the needed tables (we can still show this example but recommend *strongly* against using `*`).

Reply:
Yea makes sense.  I went back and forth on whether to recommend against using * but will add that in.